OpenCorePkg.dsc

```diff
@@ -160,7 +160,7 @@
   UefiBootServicesTableLib|MdePkg/Library/UefiBootServicesTableLib/UefiBootServicesTableLib.inf
   UefiBootManagerLib|MdeModulePkg/Library/UefiBootManagerLib/UefiBootManagerLib.inf
   UefiDriverEntryPoint|OpenCorePkg/Library/OcDriverEntryPoint/UefiDriverEntryPoint.inf
-  UefiHiiServicesLib|MdeModulePkg/Library/UefiHiiServicesLib/UefiHiiServicesLib.inf
+  UefiHiiServicesLib|OpenCorePkg/Library/OcHiiServicesLib/OcHiiServicesLib.inf
```

Comment:
Suspect this is another change which makes sense anyway. Resolves same issue as in 13a1c62446a9bea66a4ff11771f166b7ee31464f - namely that the EDK 2 HiiServicesLib asserts when officially _optional_ protocols are not present.